typescript/infra/helm/helloworld-kathy/templates/_helpers.tpl

```diff
@@ -5,6 +5,24 @@ Expand the name of the chart.
 {{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
```

Comment:
this is a super common snippet that helm charts use (as in, I didn't write this bit, it's a widely used rune). It's recommended as a best practice whenever you're using names for resources that aren't hardcoded because otherwise you may accidentally violate the k8s naming requirements. So this isn't in response to an issue or anything, just now that the names of resources aren't hardcoded I added it in